integrationTests/basic/jwt_auth.test.js

```diff
@@ -222,15 +222,15 @@ describe('jwt auth', () => {
 
         it('successfully authenticates with `jwtGithubAudience` set to `sigstore`', async () => {
             when(core.getInput)
-                .calledWith('role')
+                .calledWith('role', expect.anything())
                 .mockReturnValueOnce('default-sigstore');
 
             when(core.getInput)
-                .calledWith('jwtGithubAudience')
+                .calledWith('jwtGithubAudience', expect.anything())
                 .mockReturnValueOnce('sigstore');
 
             when(core.getIDToken)
-                .calledWith()
+                .calledWith(expect.anything())
```

Comment:
Why are some of these using `expect.anything()` and others use `undefined`?

Reply:
Annoying, `expect.anything()` doesn't seem to accept `undefined`, so in the cases where the function was getting called with `undefined`, we needed to explicitly use that.